syncd/syncd_pfc_watchdog.cpp

```diff
@@ -0,0 +1,260 @@
+#include "syncd_pfc_watchdog.h"
```

Comment:
this a generic port queue counter collection thread. we can rename it to syncd_flex_counters.cpp